index.js

```diff
@@ -314,7 +314,7 @@ exports.evm2wast = function (evmCode, opts = {
 
     // creates a stack trace
     if (opts.stackTrace) {
```

Comment:
This block probably should move before the switch statement, but the `sp` is only calculated after the switch statement. Will need to investigate this later, it shouldn't block this PR.